clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -145,11 +145,14 @@ export class SettingsService {
    * @param dialectType name of the cobol dialect type
    * @returns a list of local path
    */
-  public static getCopybookLocalPath(
+  public static async getCopybookLocalPath(
```

Comment:
There are mocks in `CopybookMessageHandlerTest.spec.ts` and `CopybookResolveURITest.spec.ts` that should be adjusted to return Promises as well. The same goes for `getCopybookExtension`.